win32ss/user/user32/windows/draw.c

```diff
@@ -759,7 +759,7 @@ static BOOL UITOOLS95_DFC_ButtonCheckRadio(HDC dc, LPRECT r, UINT uFlags, BOOL R
         {
             TCHAR Check = (Radio) ? 'i' : 'b';
 
-            SetTextColor(dc, GetSysColor(COLOR_WINDOWTEXT));
+            SetTextColor(dc, GetSysColor((uFlags & DFCS_INACTIVE) ? COLOR_BTNSHADOW : COLOR_WINDOWTEXT));
```

Comment:
Should it really be COLOR_BTNSHADOW or COLOR_GRAYTEXT instead?

Reply:
Likely COLOR_GRAYTEXT

Reply:
Using a manual test similar to https://learn.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-setsyscolors#remarks
for trying to change the RGB color associated to COLOR_BTNSHADOW or COLOR_GRAYTEXT, may help determining the difference.